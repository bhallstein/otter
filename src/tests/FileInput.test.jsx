import React from 'react';
import FileInput from '../generic/FileInput';
import TestRenderer from 'react-test-renderer';

it('renders correctly', () => {
  // mocks
  const value = '';

  const tree = TestRenderer
    .create(<FileInput value={value}></FileInput>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});