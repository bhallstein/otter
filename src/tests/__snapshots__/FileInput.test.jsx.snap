// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="di mr3 my3 w-40"
>
  <label
    htmlFor="file"
  >
    Selected file: 
    
    <span
      className="mx4 pa2 ba color-border-grey-4 br10 cursor-pointer"
    >
      Change
    </span>
  </label>
</div>
`;
